ccd-definition/FixedLists.json

```diff
@@ -326,7 +326,7 @@
   {
     "LiveFrom": "01/01/2017",
     "ID": "Consent",
-    "ListElementCode": "agree",
+    "ListElementCode": "AGREE",
```

Comment:
Changing this have higher price. Let's consider later whether we need to make that change.

Reply:
So should i revert this change?